setup.py

```diff
@@ -51,9 +51,14 @@ def read_requirements(filename):
     "boto3>=1.26.131,<2.0",
     "cloudpickle==2.2.1",
     "google-pasta",
+    "google-api-core==2.8.0",
+    "google-auth==2.6.6",
```

Comment:
Hi @knikure, is there a reason to add these google dependencies? If so, I would be more in favor to define possible ranges values instead of strict version. 

`google-auth` is now at version 2.18.1 for example: https://pypi.org/project/google-auth/#history. That means any project that uses sagemaker-sdk with another version of these google packages will have problem